From 1403303e77ce0c5a4d4c52d6a3b2a820f47241ee Mon Sep 17 00:00:00 2001
From: Emmanuele Bassi <ebassi@gnome.org>
Date: Fri, 30 Apr 2021 12:15:25 +0100
Subject: [PATCH] Remove redeclaration of khronos_uintptr_t

The type is being redeclared because I didn't see the original
declaration when I wrote 144cbc9325250081f2eb584ca5deb13aaf1c2433.

Fixes: #249
---
 src/gen_dispatch.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/gen_dispatch.py b/src/gen_dispatch.py
index b686824..3daad84 100755
--- a/src/gen_dispatch.py
+++ b/src/gen_dispatch.py
@@ -535,7 +535,6 @@ def write_header(self, out_file):
             self.outln('    KHRONOS_TRUE  = 1,')
             self.outln('    KHRONOS_BOOLEAN_ENUM_FORCE_SIZE = KHRONOS_MAX_ENUM')
             self.outln('} khronos_boolean_enum_t;')
-            self.outln('typedef uintptr_t khronos_uintptr_t;')
 
         if self.target == "glx":
             self.outln('#include <X11/Xlib.h>')
